ape_tokens/converters.py

```diff
@@ -26,8 +26,10 @@ def is_convertible(self, value: Any) -> bool:
         provider = self.network_manager.active_provider
         if not provider:
             raise ConversionError("Must be connected to a provider to use the token converter.")
-
-        tokens = self.manager.get_tokens(chain_id=provider.network.chain_id)
+        try:
+            tokens = self.manager.get_tokens(chain_id=provider.network.chain_id)
+        except:
```

Comment:
```suggestion
        except ValueError as err:
```

Let the other exceptions blow up for now.